sdk/core/Azure.Core/src/Pipeline/HttpClientTransport.cs

```diff
@@ -27,8 +28,14 @@ public class HttpClientTransport : HttpPipelineTransport
         /// Creates a new <see cref="HttpClientTransport"/> instance using default configuration.
         /// </summary>
         public HttpClientTransport() : this(CreateDefaultClient())
-        {
-        }
+        { }
+
+        /// <summary>
+        /// Creates a new <see cref="HttpClientTransport"/> instance using default configuration.
+        /// </summary>
+        /// <param name="options">The <see cref="HttpPipelineTransportOptions"/> that to configure the behavior of the transport.</param>
+        public HttpClientTransport(HttpPipelineTransportOptions? options = null) : this(CreateDefaultClient(options))
```

Comment:
Let's make specific ctors private for now and see if anyone wants them.

Reply:
ConfidentialLedger needs one to pass options to.

Reply:
Why not to HttpPipelineTransport.Create?